stripe/src/main/java/com/stripe/android/paymentsheet/BaseAddCardFragment.kt

```diff
@@ -305,6 +307,16 @@ internal abstract class BaseAddCardFragment(
         cardMultilineWidget.postalCodeErrorListener = null
     }
 
+    private fun populateFieldsFromNewCard() {
+        sheetViewModel.newCard?.let { it ->
+            it.paymentMethodCreateParams.let { params ->
+                cardMultilineWidget.populateFromParams(params)
+                billingAddressView.populateFromParams(params)
+                saveCardCheckbox.isChecked = false
```

Comment:
I think this should be `saveCardCheckbox.isChecked = it.shouldSavePaymentMethod`